pkgs/development/libraries/libical/default.nix

```
@@ -12,6 +12,7 @@
 , libical
 , python3
 , tzdata
+, fixDarwinDylibNames
 , introspectionSupport ? stdenv.buildPlatform == stdenv.hostPlatform
 , gobject-introspection ? null
 , vala ? null
```

```suggestion
, gobject-introspection
, vala
```

and the assert can be removed.